### PR TITLE
Add The Gentle Men FA news post

### PR DESCRIPTION
**Pull request by pishifat**

### CODE REVIEW

File: news/2021-02-24-new-featured-artist-the-gentle-men.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+![](https://assets.ppy.sh/artists/142/header.jpg)
+
+Lyricist/vocalist Charlie White (a.k.a. Twitch streamer [MoistCr1TiKaL](https://www.twitch.tv/moistcr1tikal)) and seasoned musical expert Troy McKubre are **The Gentle Men** — a pair responsible for producing rock-focused audio experiences that blur the line between high effort jokes and legitimately good music.
+
+We managed to snag **all 6** of their currently released songs, each with a slightly different genre focus, like *Obsession*'s tilt towards electronic pop and *I Am Truth*'s metalcore tone.
+
+"Are these songs worth mapping?" you may be asking yourself. Charlie played through some maps of **The Gentle Men** songs recently, so if you're looking for a sample, give his video and these previews a peek:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/YqEF_PHtEKw" frameborder="0" allowfullscreen></iframe>
```
Comment (Nivalyx) [inline news/2021-02-24-new-featured-artist-the-gentle-men.md:17]: i think you meant to use `allowfullscreen="true"` here

Comment (Ephemeralis) [reply news/2021-02-24-new-featured-artist-the-gentle-men.md:17]: youtube embed has used name presence over value for several years at this point i believe

Comment (pishifat) [reply news/2021-02-24-new-featured-artist-the-gentle-men.md:17]: current is correct according to other youtube embeds in news posts

Comment (Nivalyx) [reply news/2021-02-24-new-featured-artist-the-gentle-men.md:17]: ah, i see

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**